schemas/test/en/test_theme_ukhsa_ons.json

```diff
@@ -0,0 +1,75 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "Test Winter Coronavirus Infection Study",
```

Comment:
I would remove this from the title. Just name it after UKHSA. The survey isn't important, they may run many.